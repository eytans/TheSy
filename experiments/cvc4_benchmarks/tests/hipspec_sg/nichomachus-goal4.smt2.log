05:52:46 [WARN] Collecting statistics
05:52:46 [WARN]  ex is: zero
05:52:46 [WARN]  ex is: (succ zero)
05:52:46 [WARN]  ex is: (succ (succ zero))
05:52:46 [WARN] Running exploration without proof mode, but goals were given
05:52:46 [WARN] Proof mode: true
05:52:46 [WARN]  ex is: zero
05:52:46 [WARN]  ex is: (succ zero)
05:52:46 [WARN]  ex is: (succ (succ zero))
05:52:46 [WARN] Running exploration without proof mode, but goals were given
05:52:46 [WARN] Proof mode: true
05:52:46 [WARN] generalized case_split proof as: (mult ?ts_ph_Nat_1 (plus ?ts_ph_Nat_0 ?ts_ph_Nat_0)) -- (mult ?ts_ph_Nat_0 (plus ?ts_ph_Nat_1 ?ts_ph_Nat_1))
05:52:46 [WARN] proved: (mult ?ts_ph_Nat_1 (plus ?ts_ph_Nat_0 ?ts_ph_Nat_0)) => (mult ?ts_ph_Nat_0 (plus ?ts_ph_Nat_1 ?ts_ph_Nat_1))
05:52:46 [WARN] proved: (mult ?ts_ph_Nat_0 (plus ?ts_ph_Nat_1 ?ts_ph_Nat_1)) => (mult ?ts_ph_Nat_1 (plus ?ts_ph_Nat_0 ?ts_ph_Nat_0))
05:52:46 [WARN] generalized case_split proof as: (mult ?ts_ph_Nat_1 (plus ?ts_ph_Nat_0 ?ts_ph_Nat_2)) -- (plus (mult ?ts_ph_Nat_0 ?ts_ph_Nat_1) (mult ?ts_ph_Nat_2 ?ts_ph_Nat_1))
05:52:46 [WARN] proved: (mult ?ts_ph_Nat_1 (plus ?ts_ph_Nat_0 ?ts_ph_Nat_2)) => (plus (mult ?ts_ph_Nat_0 ?ts_ph_Nat_1) (mult ?ts_ph_Nat_2 ?ts_ph_Nat_1))
05:52:46 [WARN] proved: (plus (mult ?ts_ph_Nat_0 ?ts_ph_Nat_1) (mult ?ts_ph_Nat_2 ?ts_ph_Nat_1)) => (mult ?ts_ph_Nat_1 (plus ?ts_ph_Nat_0 ?ts_ph_Nat_2))
05:52:46 [WARN] Found all lemmas
