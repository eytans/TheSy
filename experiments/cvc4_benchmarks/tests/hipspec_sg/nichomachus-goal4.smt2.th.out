05:52:46 [0m[33m[WARN] [0mCollecting statistics
(plus zero ?n) => ?n
(plus (succ ?n) ?m) => (succ (plus ?n ?m))
(succ (plus ?n ?m)) => (plus (succ ?n) ?m)
(mult zero ?n) => zero
(mult (succ ?n) ?m) => (plus (mult ?n ?m) ?m)
(plus (mult ?n ?m) ?m) => (mult (succ ?n) ?m)
(tri zero) => zero
zero => (tri zero)
(tri (succ ?n)) => (plus (tri ?n) (succ ?n))
(plus (tri ?n) (succ ?n)) => (tri (succ ?n))
(cubes zero) => zero
zero => (cubes zero)
(cubes (succ ?n)) => (plus (cubes ?n) (mult (succ ?n) (mult (succ ?n) (succ ?n))))
(plus (cubes ?n) (mult (succ ?n) (mult (succ ?n) (succ ?n)))) => (cubes (succ ?n))
(plus ?n ?m) => (plus ?m ?n)
(plus ?m ?n) => (plus ?n ?m)
(plus ?x (plus ?y ?z)) => (plus (plus ?y ?x) ?z)
(plus (plus ?y ?x) ?z) => (plus ?x (plus ?y ?z))
(mult ?n ?m) => (mult ?m ?n)
(mult ?m ?n) => (mult ?n ?m)
05:52:46 [0m[33m[WARN] [0m ex is: zero
05:52:46 [0m[33m[WARN] [0m ex is: (succ zero)
05:52:46 [0m[33m[WARN] [0m ex is: (succ (succ zero))
05:52:46 [0m[33m[WARN] [0mRunning exploration without proof mode, but goals were given
05:52:46 [0m[33m[WARN] [0mProof mode: true
05:52:46 [0m[33m[WARN] [0m ex is: zero
05:52:46 [0m[33m[WARN] [0m ex is: (succ zero)
05:52:46 [0m[33m[WARN] [0m ex is: (succ (succ zero))
05:52:46 [0m[33m[WARN] [0mRunning exploration without proof mode, but goals were given
05:52:46 [0m[33m[WARN] [0mProof mode: true
05:52:46 [0m[33m[WARN] [0mgeneralized case_split proof as: (mult ?ts_ph_Nat_1 (plus ?ts_ph_Nat_0 ?ts_ph_Nat_0)) -- (mult ?ts_ph_Nat_0 (plus ?ts_ph_Nat_1 ?ts_ph_Nat_1))
05:52:46 [0m[33m[WARN] [0mproved: (mult ?ts_ph_Nat_1 (plus ?ts_ph_Nat_0 ?ts_ph_Nat_0)) => (mult ?ts_ph_Nat_0 (plus ?ts_ph_Nat_1 ?ts_ph_Nat_1))
05:52:46 [0m[33m[WARN] [0mproved: (mult ?ts_ph_Nat_0 (plus ?ts_ph_Nat_1 ?ts_ph_Nat_1)) => (mult ?ts_ph_Nat_1 (plus ?ts_ph_Nat_0 ?ts_ph_Nat_0))
05:52:46 [0m[33m[WARN] [0mgeneralized case_split proof as: (mult ?ts_ph_Nat_1 (plus ?ts_ph_Nat_0 ?ts_ph_Nat_2)) -- (plus (mult ?ts_ph_Nat_0 ?ts_ph_Nat_1) (mult ?ts_ph_Nat_2 ?ts_ph_Nat_1))
05:52:46 [0m[33m[WARN] [0mproved: (mult ?ts_ph_Nat_1 (plus ?ts_ph_Nat_0 ?ts_ph_Nat_2)) => (plus (mult ?ts_ph_Nat_0 ?ts_ph_Nat_1) (mult ?ts_ph_Nat_2 ?ts_ph_Nat_1))
05:52:46 [0m[33m[WARN] [0mproved: (plus (mult ?ts_ph_Nat_0 ?ts_ph_Nat_1) (mult ?ts_ph_Nat_2 ?ts_ph_Nat_1)) => (mult ?ts_ph_Nat_1 (plus ?ts_ph_Nat_0 ?ts_ph_Nat_2))
05:52:46 [0m[33m[WARN] [0mFound all lemmas
done in 353
